packages/palette/src/elements/DonutChart/DonutChart.tsx

```diff
@@ -0,0 +1,164 @@
+import { interpolate } from "d3-interpolate"
+import { arc as d3_arc, pie as d3_pie } from "d3-shape"
+import React, { useRef, useState } from "react"
+import { Spring } from "react-spring"
+import styled from "styled-components"
+import { color, space } from "../../helpers"
+import { Color } from "../../Theme"
+import { Box } from "../Box"
+import { ChartHoverTooltip } from "../DataVis/ChartHoverTooltip"
+import { coerceTooltip } from "../DataVis/ChartTooltip"
+import { ProvideMousePosition } from "../DataVis/MousePositionContext"
+import { ChartProps } from "../DataVis/utils/SharedTypes"
+import { useGetWrapperWidth } from "../DataVis/utils/useGetWrapperWidth"
+import { useHasEnteredViewport } from "../DataVis/utils/useHasEnteredViewPort"
+import { Sans } from "../Typography"
+
+const colors: Color[] = ["black10", "black30", "black60"]
+
+export interface DonutChartProps extends ChartProps {
+  margin?: number
+}
+
+/**
+ * DonutChart is a component that displays data points with donut shaped arcs.
+ * Good for illustrating numerical proportions.
+ */
+export const DonutChart: React.FC<DonutChartProps> = ({
+  points,
+  margin = space(3),
+}: DonutChartProps) => {
+  const [hoverIndex, setHoverIndex] = useState(-1)
+  const [rest, setRest] = useState(false)
+
+  const wrapperRef = useRef<HTMLDivElement>(null)
+
+  const hasEnteredViewport = useHasEnteredViewport(wrapperRef)
+
+  const width = useGetWrapperWidth(wrapperRef)
+
+  const donutWidth = width * 0.12
```

Comment:
According to the spec `donutWidth` should be based on screen size. 30px on mobile and 40px on desktop. I noticed this because the donuts looked a bit fatter than the spec to me.

I personally think slightly fatter is fine, but we should clear it with @owendodd

Reply:
Sorry I should have mentioned it in the PR. We had a conversation on slack about it: https://artsy.slack.com/archives/C9XJKPY9W/p1559851503006000

Happy to fine tune that ratio if you think it needs to be thinner.

Reply:
Nah it's good 👍